distributed/protocol/serialize.py

```diff
@@ -195,6 +202,14 @@ def deserialize(self):
         frames = decompress(self.header, self.frames)
         return deserialize(self.header, frames)
 
+    def __eq__(self, other):
+        return (isinstance(other, Serialized) and
+                other.header == self.header and
+                other.frames == self.frames)
+
+    def __ne__(self, other):
+        return not (self == other)
```

Comment:
Just for the unit tests. It has no use in the rest of the code base. `__hash__` isn't necessary as we don't use those objects as keys in associative containers.